config/plugins/visualizations/aequatus/templates/aequatus.mako

```diff
@@ -21,11 +21,8 @@
         ${h.javascript_link( app_root + "aequatus-vis/scripts/jquery/js/jquery-migrate-1.4.1.min.js" )}
         ${h.javascript_link( app_root + "aequatus-vis/scripts/scriptaculous/prototype.js" )}
 
-
-## install shared libraries
-        ${h.js( 'libs/d3')}
-
 ## aequatus-vis
+        ${h.javascript_link( app_root + "aequatus-vis/scripts/d3.min.js" )}
```

Comment:
Hi Dannon,

I think you forgot to add this d3.min.js

Thanks

Anil

Reply:
@anilthanki It should be there, let me doublecheck.